Automattic/wp-calypso
# Create google my business invite

This PR adds the UX elements for the Google My Business nudge as we don't have any other screens to add the actions yet.

### Specs
See p989we-co-p2:

The Traffic tab in the Stats page will now show the following nudge to users identified in the signup survey:

Desktop
![nudge](https://user-images.githubusercontent.com/2810519/35992902-877005c4-0cc0-11e8-9225-4deb9878d07f.png)

Mobile
![mobile nudge](https://user-images.githubusercontent.com/2810519/35992904-897a606c-0cc0-11e8-8a73-29dea3708051.png)

### Testing Instructions
- Boot this branch locally or use the calypso.live link below
- Create a site with a `promote` siteGoals if you don't have one. This can be done from the `http://calypso.localhost:3000/start/about` page by selecting at least the "Promote your business, skills, organization, or events" option.
- Navigate to this site stats page `/stats/day/:site_slug`
- Notice the default stat page
- Upgrade your site plan to business or premium
- Go back to the stats page
- Notice the GMB nudge
- Check that it displays properly for different types of screens and on various browsers.

### Reviews
